test/single-workspace-tests/extension/utilities/utilityFunctions.test.ts

```diff
@@ -0,0 +1,25 @@
+import * as chai from 'chai';
```

Comment:
please switch to an intend of 4 spaces (to adhere with the rest of the code)
(in the file)

Reply:
Dang! I did in the other PR (there's gonna be a conflict, btw, as this file is introduced in both, so once #336 is merged in, I'll re-work this so it merges fine).